ts/Dashboards/Plugins/HighchartsComponent.ts

```diff
@@ -629,20 +645,37 @@ class HighchartsComponent extends Component {
                     return true;
                 });
 
+            // create empty series for OHLC data
```

Comment:
It doesn't have to be only OHLC- area range, x-range also works.
I would change the comment to something more generic.